js/lesson4/tutorial.md

```diff
@@ -289,7 +289,7 @@ function retrieveGenres() {
 
 > You can use `<title>` to display a humanly readable format of the Genre
 
-As you can see from the console, the resulting objects are returned inside an Array. We want to iterate over the list using the `$.each( )` function and add each item to the `#genres` list, as a **list item**. As we need to have access to the `key` as well, we can set that as the list item's `id`.
+As you can see from the console, the resulting objects are returned inside an Array. We want to iterate over the list using the `forEach( )` function and add each item to the `#genres` list, as a **list item**. As we need to have access to the `key` as well, we can set that as the list item's `id`.
```

Comment:
We should probably clarify here that we mean the native method. How about we change the following
> using the `forEach( )` function
to something like
> using the native Array `forEach( )` method